pkg/security/certnames/locator.go

```diff
@@ -24,7 +24,7 @@ type Locator struct {
 
 // MakeLocator initializes a Locator.
 func MakeLocator(certsDir string) Locator {
-	return Locator{certsDir: certsDir}
+	return Locator{certsDir: os.ExpandEnv(certsDir)}
```

Comment:
This doesn't seem right. The expansion should be done by the shell, not crdb.

Where is this $HOME default defined?

Reply:
oh I found it - base/config.go: `DefaultCertsDirectory = "${HOME}/.cockroach-certs"`